_docs/technical/2017-01-04-updater.md

```diff
@@ -123,41 +147,69 @@ for more information.
 ## Update Proposal Approval
 
 A very important part of implementation of the update mechanism is
-the part that works with genesis blocks for epochs and applies updates.
+the part that works with genesis blocks for epochs.
 This logic resides
-[in this well-documented function](https://github.com/input-output-hk/cardano-sl/blob/22360aa45e5dd82d0c87872d8530217fc3d08f4a/src/Pos/Update/Poll/Logic/Softfork.hs#L67).
+[in this well-documented function](https://github.com/input-output-hk/cardano-sl/blob/6b5eda44e5942599a9781e5ad3f51eb820665b83/src/Pos/Update/Poll/Logic/Softfork.hs#L68).
 We explain terminology related to this process below.
 
 ### Threshold
 
 `softforkResolutionThreshold` is a threshold (referred to as «threshold»
-in the code) which, for an update proposal with version `v`, says ???
+in the code), which allows to define stable stake for each stakeholder
+as their stake `k` blocks ago. See more detailed description in
```

Comment:
I don't understand what's written here. How can threshold allow to define stable stake?
Suppose there is a block version `X`. And there are block with version `X` created in slots `S` (`S` is a set of slots). If total relative stake of leaders of all slots in `S` is ≥ `X`, then `X` becomes adopted.